packages/vscode-extension/src/controls/webviewPanel.ts

```diff
@@ -33,7 +33,7 @@ export class WebviewPanel {
   private panelType: PanelType = PanelType.SampleGallery;
   private disposables: vscode.Disposable[] = [];
 
-  public static createOrShow(panelType: PanelType, isToSide?: boolean) {
+  public static createOrShow(panelType: PanelType, isToSide?: boolean, args?: any[]) {
```

Comment:
minor: merging `isToSide` and `args` might be better because `isToSide` can be derived from `args[0]`.

Reply:
Done